### PR TITLE
Add KickEX

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Aren't you tired Aries? It's 00:30 on a Wednesday :sleeping:

**Comment by 40504** [comment]
> Aren't you tired Aries? It's 00:30 on a Wednesday 😴

A little :)

File: img/k/kickex.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...044-.086 0-.044.043-.044.043-.087 0 0 0-.043.043-.043 0-.043 0-.043.043-.087v-.043c0-.043.043-.086.043-.13v-.173c0-.043.044-.086.044-.13v-.129c0-.043 0-.13.043-.173v-.389l-.043-17.948c0-1.73-1.384-3.071-3.113-3.071s-3.112 1.384-3.069 3.114l.043 12.585L13.271 22.62c1.254-1.773 2.68-3.503 4.28-5.06 7.305-7.31 17.031-11.374 27.406-11.374s20.1 4.022 27.406 11.374c1.6 1.6 3.026 3.287 4.28 5.103-.044 0-.087.043-.13.043L54.467 35.334l.043-12.585c0-1.73-1.383-3.114-3.07-3.114-1.685 0-3.111 1.384-3.111 3.07l-.044 17.949v.389c0 .043 0 .13.044.173v.13c0 .043.043.086.043.13v.172c0 .044.043.087.043.13v.043c0 .043 0 .043.043.087 0 0 0 .043.043.043 0 .043.044.043.044.086 0 .044.043.087.043.087l.043.043v.043s0 .044.043.044c.043.043.043.086.087.13 0 .043.043.043.086.086l.044.043.043.043.043.044.043.043c.043.043.087.13.13.173l.043.043.13.13.043.043c.043.043.086.043.086.087.044 0 .044.043.087.043.043.043.087.086.173.086.043 0 .043.044.086.044 0 0 .044 0 .044.043l15.518 8.39c.476.26.951.39 1.47.39 1.08 0 2.161-.606 2.723-1.644a3.082 3.082 0 0 0-1.253-4.195l-10.591-5.709 21.873-12.542c.086-.043.13-.087.217-.13a38.776 38.776 0 0 1 3.933 16.997c0 10.466-4.064 20.197-11.369 27.506Z" fill="#0CC466"/></svg>
```
Comment (Carlgo11) [inline img/k/kickex.com.svg]: There's a few problems with the SVG that's other of our control.

Basically, the SVG that KickEX uses is one of the worst vector graphics I've ever seen.
I don't understand how they managed to fail that bad but if you zoom in on the arrow head you can see something which looks more like the Gadsden flag's snake than the head of an arrow.

Actually... let me present to you the Gadsden flag by KickEX. A story in four parts:
First we copy the head of one of the arrows:
screenshot_1643154752

Then we paste it:
screenshot_1643155782

Add a tongue:
screenshot_1643155853

A voilà! A snake which kindly requests you change trajectory of feet to avoid collision.


TLDR; we can't use this SVG because it would make vector graphic designers cry.

I did some googling to try and find a good PNG to replace the SVG with but couldn't find one. Maybe you have better luck? If not then I can take the JPEG they use on their twitter and crop out the other white space.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/k/kickex.com.svg (hunk 1 above)
Comment (Carlgo11) [reply img/k/kickex.com.svg]: I wonder how much I could earn if I sold the snek as an NFT :thinking: 
Maybe the KickEX staff are interested https://twitter.com/carlgo11/status/1486138922019217409

Comment (40504) [reply img/k/kickex.com.svg]: 🤣🤣🤣 👍 🧐

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]
Great work @40504! Doing the work that a "next-generation cryptocurrency exchange [...] [p]owered by in-house software developed from scratch" couldn't manage :fist_raised:

Let's hope their software is better developed than their logo.
